bats/tests/helpers/utils.bash

```diff
@@ -105,6 +105,9 @@ join_map() {
     echo "$result"
 }
 
+# Run jq on the current $output
+# Note that you should probably be using `run --separate-stderr` to avoid issues
+# with trying to parse stderr.
```

Comment:
It was at first not clear to me that the comment recommend to run the previous command with `--separate-stderr`, and not the `jq_output` command itself, like

```
run --separate-stderr jq_output .foo
```

This could probably be improved even more:

```suggestion
# Note that you should probably be using `run --separate-stderr` when capturing $output
# to avoid issues with trying to parse stderr. Alternatively use run_without_error().
```